### ISSUE TITLE
During yarn install, getting network connection error

### ISSUE THREAD

**Issue by pkim050** [mikaelbr/node-notifier#414]
yarn install
[1/4] Resolving packages...
[2/4] Fetching packages...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
error An unexpected error occurred: "https://registry.yarnpkg.com/node-notifier/-/node-notifier-8.0.1.tgz: read ECONNRESET".
info If you think this is a bug, please open a bug report with the information provided in "/opt/current/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
The command '/bin/sh -c yarn install' returned a non-zero code: 1
Cleaning up project directory and file based variables
ERROR: Job failed: exit status 1

After couple of hours of trouble shooting, I am not getting aborted instead of read ECONNRESET.
Attempts:
Extending timeout network by 100000 -> yarn install --network-timeout 100000
Running yarn cache clean before yarn install
Running proxy and https-proxy to be removed before yarn install
Tried upgrading node-notifier package to 8.0.2 from 8.0.1
Tried removing yarn.lock
Tried running the pipeline for non production with the same exact code for Dockerfile, package.json, and yarn.lock, worked for non prod but still doesnt work on prod pipeline

Definitely need some help and suggestions.

**Comment by darapuk** [comment]
It's likely your company VPN, can you try turning off your vpn and see if it works? We are seeing similar issues as this package is being flagged for containing a virus for some reason

**Comment by pkim050** [comment]
Update:
So we went to check out our build servers and used wget https://registry.yarnpkg.com/node-notifier/-/node-notifier-8.0.1.tgz.
On our Non Production build server, we were able to download the package while on Production build server, it hangs during the fetching process.
So it is definitely a connectivity issue on our side (not sure what but this has nothing to do for this repository).

Going to go ahead and close this thread.

Thank you @darapuk!
